File: Programming EF Code First/chapter6/Chapter6/Chapter 6 VB/Fluent/Model/Lodging.vb
Namespace Model
  Public Class Lodging
	Public Property LodgingId() As Integer
	Public Property Name() As String
	Public Property Owner() As String
	Public Property MilesFromNearestAirport() As Decimal

	Public Property DestinationId() As Integer
	Public Property Destination() As Destination
	Public Property InternetSpecials() As List(Of InternetSpecial)
	Public Property PrimaryContactId() As Integer?
	Public Property PrimaryContact() As Person
	Public Property SecondaryContactId() As Integer?
	Public Property SecondaryContact() As Person
  End Class
End Namespace

File: Programming EF Code First/chapter6/Chapter6/BAGA_FluentAPI/Model/Lodging.cs
using System.Collections.Generic;
using System;

namespace Model
{
  public class Lodging
  {
    public int LodgingId { get; set; }
    public string Name { get; set; }
    public string Owner { get; set; }
    public decimal MilesFromNearestAirport { get; set; }

    public int DestinationId { get; set; }
    public Destination Destination { get; set; }
    public List<InternetSpecial> InternetSpecials { get; set; }
    public Nullable<int> PrimaryContactId { get; set; }
    public Person PrimaryContact { get; set; }
    public Nullable<int> SecondaryContactId { get; set; }
    public Person SecondaryContact { get; set; }
  }
}